db/migrate/20230106114221_update_default_cohort_for_ehco.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateDefaultCohortForEhco < ActiveRecord::Migration[6.1]
+  def change
+    ehco_course = Course.find_by_code(code: :EHCO)
```

Comment:
Very minor but there is a scope for this: `Course.ehco` up to you whether included